sdk/cosmosdb/cosmos/test/internal/unit/sasToken.spec.ts

```diff
@@ -1,12 +1,10 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 import assert from "assert";
-import { CosmosClient } from "../../../dist-esm/";
+import { CosmosClient } from "@azure/cosmos";
```

Comment:
The @azure/cosmos import form should not be used in tests, only in samples.

Reply:
Thanks. Fixed the imports in the tests